src/openmrs-esm-error-handling.ts

```diff
@@ -1,2 +1,13 @@
-// eslint-disable-next-line
-console.log("hello world");
+// import { showToast } from '@openmrs/esm-styleguide'
+
+export function handleApiError() {
+  return incomingResponseErr => {
```

Comment:
incomingResponseErr will be an error everytime?

Reply:
Yes. See https://github.com/openmrs/openmrs-esm-api/blob/master/src/openmrs-fetch.ts#L112.

Although maybe it would be better to still check if its an error, and if it's not then wrap it in an error?

Reply:
I'll merge this for now to unblock you in your work on this. We can update handleApiError to be more robust in a future PR if we want to.

Reply:
I think it is ok to let it like that. It is not our interest to let anyone use api wihtout using openmrsFetch anyway, so I think i would be a idea to check it only to make throw something that will make the developer aware of that.